graphql_client_codegen/src/shared.rs

```diff
@@ -6,6 +6,19 @@ use proc_macro2::{Ident, Span, TokenStream};
 use query::QueryContext;
 use selection::*;
 
+fn format_available_fields(fields: &[GqlObjectField]) -> String {
+    fields
+        .iter()
+        .map(|ref field| &field.name)
+        .fold(String::new(), |mut acc, item| {
+            acc.push_str(item);
+            acc.push_str(", ");
```

Comment:
I think `Itertools::format` would just be better.

```rust
fields.iter().map(|ref field| &field.name).format("`, `")
```

This keeps the field names enclosed in "`", but leaves the commas without markup.